embed/templates/scripts/run_grafana.sh.tpl

```diff
@@ -6,6 +6,15 @@ set -e
 DEPLOY_DIR={{.DeployDir}}
 cd "${DEPLOY_DIR}" || exit 1
 
+# try to compatible with tidb-lightning's dashboard,
+# the tidb-lightning dashboard's uid is not set,
+# after the cluster renamed, the old dashboard will not be replaced with the newly one.
+# and Grafana use the unique uid as indepent dashaboard.
+# this script was stolen from https://stackoverflow.com/a/13844351
+if [[ -f dashboards/lightning.json ]]; then
+    grep -qxF uid dashboards/lightning.json || sed -i '$s/}/,\n"uid":"t5PjsKUGz"}/' dashboards/lightning.json
```

Comment:
So could you remove these lines and open a PR on https://github.com/pingcap/tidb-lightning/blob/master/metrics/grafana/lightning.json?

Reply:
So for the Grafana package that has been released, are we not going to fix it, to keep the code simple?

Reply:
Sorry for my mistake: https://github.com/PingCAP-QE/ci/blob/jenkins-pipelines/jenkins/pipelines/cd/tiup/grafana-tiup-mirror-update-test.groovy#L95

We should change BR instead...

Reply:
> So for the Grafana package that has been released, are we not going to fix it, to keep the code simple?

No, we shouldn't change codesthat not belong to TiUP. (I think the xxx.json in other repos can be treated as code). The user can upgrade cluster if he really want to fix this.